scalafix-tests/src/it/scala/scalafix/tests/URLConfiguration.scala

```diff
@@ -20,4 +20,37 @@ class URLConfiguration
        |
     """.stripMargin
   DiffTest("url", code, "UrlRewrite.stat").foreach(runDiffTest)
+
+  val githubURL =
+    "github:scalacenter/scalafix/1.0"
```

Comment:
yep, I realized it while writing it as well. I'll add a unit test and remove this.